myproject/supermarket_manager/views.py

```diff
@@ -1,15 +1,21 @@
 from django.http import JsonResponse
 from django.db.utils import IntegrityError
 from django.views.decorators.csrf import csrf_exempt
-from .models import (Role, Permission, Role_Permission)
+from .models import (Role, Permission, Role_Permission, Account)
+import re
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth import authenticate, login
```

Comment:
I haven't seen you use these modules yet.

Reply:
Done